backend/app/api/endpoints/kind/marketplace.py

```
@@ -0,0 +1,314 @@
+
+router = APIRouter()
+
+
+# ============================================================================
+# Category Management Endpoints (Admin only)
+# ============================================================================
+
+
+@router.get("/skill-categories", response_model=SkillCategoryListResponse)
+async def list_skill_categories(
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.14)

[warning] Unused function argument: `current_user`

(ARG001)